tsl/test/expected/continuous_aggs_ddl.out

```diff
@@ -1359,14 +1359,166 @@ SELECT create_hypertable('conditions', 'time');
  (34,public,conditions,t)
 (1 row)
 
+INSERT INTO conditions VALUES ( '2018-01-01 09:20:00-08', 'SFO', 55);
+INSERT INTO conditions VALUES ( '2018-01-02 09:30:00-08', 'por', 100);
+INSERT INTO conditions VALUES ( '2018-01-02 09:20:00-08', 'SFO', 65);
+INSERT INTO conditions VALUES ( '2018-01-02 09:10:00-08', 'NYC', 65);
+INSERT INTO conditions VALUES ( '2018-11-01 09:20:00-08', 'NYC', 45);
+INSERT INTO conditions VALUES ( '2018-11-01 10:40:00-08', 'NYC', 55);
+INSERT INTO conditions VALUES ( '2018-11-01 11:50:00-08', 'NYC', 65);
+INSERT INTO conditions VALUES ( '2018-11-01 12:10:00-08', 'NYC', 75);
+INSERT INTO conditions VALUES ( '2018-11-01 13:10:00-08', 'NYC', 85);
+INSERT INTO conditions VALUES ( '2018-11-02 09:20:00-08', 'NYC', 10);
+INSERT INTO conditions VALUES ( '2018-11-02 10:30:00-08', 'NYC', 20);
 CREATE MATERIALIZED VIEW conditions_daily
 WITH (timescaledb.continuous, timescaledb.materialized_only = false) AS
 SELECT location,
        time_bucket(INTERVAL '1 day', time) AS bucket,
        AVG(temperature)
   FROM conditions
-GROUP BY location, bucket;
-NOTICE:  continuous aggregate "conditions_daily" is already up-to-date
+GROUP BY location, bucket
+WITH NO DATA;
+-- Show both the columns and the view definitions to see that
+-- references are correct in the view as well.
+\d+ conditions_daily
```

Comment:
Usually, we recommend using the `test.show_columns()` and related functions to show this information since `psql` often breaks output format across PostgreSQL versions. 

I guess with our currently supported PG versions, psql is compatible. But it might still be a good idea to use our own test functions to emit info to ensure it is forward compatible.

Another reason to use the utility functions is that currently the test output here is extremely verbose and it is hard to know what to look for. It would be nice to be able to "filter" all the irrelevant information emitted by `\d+` with SELECT so that one can concentrate on the stuff that is actually important. It makes it easier to read the test output. I think it would also be possible to join the output of the columns names from all the views in the same query so that it is easy to see that they have the same names.

Reply:
Switched to use `test.show_columns()`.